trident/absorption_spectrum/absorption_spectrum.py

```diff
@@ -738,19 +746,26 @@ def _add_lines_to_spectrum(self, field_data, use_peculiar_velocity,
 
             # the total number of absorbers per transition
             n_absorbers = len(lambda_obs)
+            temperature = field_data['temperature'].d
 
-            # thermal broadening b parameter
-            thermal_b =  np.sqrt((2 * boltzmann_constant_cgs *
-                                  field_data['temperature']) /
-                                  line['atomic_mass'])
-
+            # the actual thermal width of the lines
+            if line['wavelength'].d == 2.1e9:
```

Comment:
It looks like `thermal_b` isn't used at all in this case. I would just not set it at all.